images/Dockerfile.ubuntu

```diff
@@ -10,41 +10,45 @@ RUN apt install -y software-properties-common
 RUN add-apt-repository ppa:oibaf/test
 RUN apt update
 RUN apt install -y \
-	systemd \
-        grub-pc-bin \
-        grub-efi-amd64-bin \
-        grub2 \
-        grub2-common \
-        nohang \
-	grub2-common \
-        sudo \
-	iproute2 \
-	squashfs-tools \
-	parted dracut \
-	dracut-network dracut-live tar \
-	e2fsprogs \
-	dosfstools \
-	coreutils \
-        network-manager \
-	debianutils \
-	curl \
-	openssh-server \
-        nano \
-	gawk \
-	haveged \
-	rsync \
-        linux-image-generic && apt-get clean
+    systemd \
+    grub-pc-bin \
+    grub-efi-amd64-bin \
+    grub2 \
+    grub2-common \
+    nohang \
+    grub2-common \
+    sudo \
+    iproute2 \
+    squashfs-tools \
+    parted dracut \
+    dracut-network tar \
+    e2fsprogs \
+    dosfstools \
+    coreutils \
+    debianutils \
+    curl \
+    openssh-server \
+    nano \
+    gawk \
+    haveged \
+    rsync \
+    neovim \
+    open-vm-tools \
+    netplan.io \
```

Comment:
any specific reason for netplan/wireguard?

Reply:
I used the OS file from PaletteOS that Venkat added for some earlier POC I believe. I will check with him if we need these.

Reply:
those can be added later on when creating the image for the provider, no need to add them here